remember_remember.py

```diff
@@ -0,0 +1,21 @@
+import PIL.ImageGrab
+from PIL import Image
+
+
+def get_encrypted_message(image_path: str) -> str:
+    """
+    A function that gets a path to an image and returns the message encrypted within it by the position of black pixels.
+    :param image_path:The path of the encryption image.
+    :return:The encrypted message.
+    """
+    image = PIL.Image.open(image_path, 'r')
+    pixels = image.load()
+    width, height = image.size
+    black_pixels = [(col, row) for col in range(width) for row in range(height) if pixels[col, row] == 1]
+    string = ''
```

Comment:
Variable names should be expressive and not generic. 
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0
In this case I think the variable is unnecessary.